cloud-controller-manager/do/loadbalancers.go

```diff
@@ -591,32 +585,48 @@ func (l *loadBalancers) buildLoadBalancerRequest(ctx context.Context, service *v
 		return nil, err
 	}
 
-	var tags []string
-	if l.resources.clusterID != "" {
-		tags = []string{buildK8sTag(l.resources.clusterID)}
-	}
-
 	return &godo.LoadBalancerRequest{
 		Name:                         lbName,
-		DropletIDs:                   dropletIDs,
-		Region:                       l.region,
 		SizeSlug:                     sizeSlug,
 		SizeUnit:                     sizeUnit,
 		ForwardingRules:              forwardingRules,
 		HealthCheck:                  healthCheck,
 		StickySessions:               stickySessions,
-		Tags:                         tags,
 		Algorithm:                    algorithm,
 		RedirectHttpToHttps:          redirectHTTPToHTTPS,
 		EnableProxyProtocol:          enableProxyProtocol,
 		EnableBackendKeepalive:       enableBackendKeepalive,
-		VPCUUID:                      l.resources.clusterVPCID,
 		DisableLetsEncryptDNSRecords: &disableLetsEncryptDNSRecords,
 		HTTPIdleTimeoutSeconds:       httpIdleTimeoutSeconds,
 		Firewall:                     buildFirewall(service),
 	}, nil
 }
 
+// buildLoadBalancerRequest returns a *godo.LoadBalancerRequest to balance
+// requests for service across nodes.
+func (l *loadBalancers) buildLoadBalancerRequest(ctx context.Context, service *v1.Service, nodes []*v1.Node) (*godo.LoadBalancerRequest, error) {
+	req, err := buildLoadBalancerRequest(service)
+	if err != nil {
+		return nil, err
```

Comment:
Let's wrap/annotate the errors from this method via `fmt.Errorf()` to ease distinguishing the possible error paths.

Reply:
I might just ignore this wrapping as this is in the CCM LoadBalancer interface implementation. I don't want to add a layer of verbosity in errors there if it's not required.